portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,17 +15,76 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Entity;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
+
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private Comment comment;
+
+  private String datastoreName = "Comments";
+
+  private String nameProperty = "name";
+  private String commentProperty = "comment";
+  private String timestampProperty = "timestamp";
+  private String upvotesProperty = "upvotes";
+  private String downvotesProperty = "downvotes";
```

Comment:
Since these are all constants, they can be `static`. Make sure to adjust the naming according to the [style guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)